README.md

```diff
@@ -130,6 +130,12 @@ logging:
   # regardless of the configured logging level.
   sampled_levels:
     'trace/webrequest': 0.2
+  # Set up the limiter for logging. When the rate of logs is close to the limit,
+  # it starts to increase the probability of discarding the logs of same kind.
+  # For default parameters use rate_limit: true
+  rate_limit:
```

Comment:
An alternative way of exposing this would be as a replacement of sampled_levels, perhaps as "level_rates", with a maximum rate for each level.